src/components/WorkOrder/Notes/NoteEntry.js

```diff
@@ -16,7 +16,12 @@ const NoteEntry = ({ note, time, user, userEmail }) => {
       <div className="note-info lbh-body-s">
         <NoteInfo time={time} user={user} email={userEmail} />
       </div>
-      {note}
+      {note.split('\n').map((el, i) => (
+        <span key={i}>
+          {el}
+          <br />
+        </span>
+      ))}
```

Comment:
Probably want to extract this to a utility function at some point but fine for now